SpatialGDK/Source/SpatialGDK/Private/Interop/RPCExecutor.cpp

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/RPCExecutor.h"
+
+#include "Interop/Connection/SpatialTraceEventBuilder.h"
+#include "Interop/SpatialPlayerSpawner.h"
+#include "Interop/SpatialReceiver.h"
+#include "Interop/SpatialSender.h"
+#include "Utils/RepLayoutUtils.h"
+
+namespace SpatialGDK
+{
+RPCExecutor::RPCExecutor(class USpatialNetDriver* InNetDriver)
+	: NetDriver(InNetDriver)
+{
+}
+
+bool RPCExecutor::ExecuteCommand(const FCrossServerRPCParams& Params)
+{
+	const TWeakObjectPtr<UObject> TargetObjectWeakPtr = NetDriver->PackageMap->GetObjectFromUnrealObjectRef(Params.ObjectRef);
+	if (!TargetObjectWeakPtr.IsValid())
+	{
+		return false;
+	}
+
+	UObject* TargetObject = TargetObjectWeakPtr.Get();
+	const FClassInfo& ClassInfo = NetDriver->ClassInfoManager->GetOrCreateClassInfoByObject(TargetObjectWeakPtr.Get());
+	UFunction* Function = ClassInfo.RPCs[Params.Payload.Index];
+	if (Function == nullptr)
+	{
+		return true;
```

Comment:
this seems unintuitive- if there is no function, we can process it?

Reply:
this is based on how it worked before. When the Function is a nullptr, we want to continue processing. I do agree it is unintuitive, but I didn't want to change current semantics more than necessary

Reply:
We used to return an RPC error if this happened though. What's the current behaviour if this occurs?

Reply:
I'm not sure what you could do with an error at this point. You could log that something went wrong and that it probably means something is actually wrong elsewhere. But the calling function can't actually do anything useful with a returned error here.

Reply:
Looking at the previous behaviour: the only thing we did extra is to log the failure. The rest of the behaviour is exactly the same.